### PR TITLE
update(express-session): version 1.18

### PR DESCRIPTION
**Pull request by peterblazejewicz**
- CookieOptions.partitioned
- CookieOptions.priority

https://github.com/expressjs/session/compare/v1.17.3...v1.18.0

Thanks!

/cc @SalmaGhoneim

Closes #68528

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] [Run `pnpm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

### CODE REVIEW

File: types/express-session/index.d.ts [hunk 1]
```diff
@@ -222,6 +222,32 @@ declare namespace session {
...
+         * - `'medium'` will set the `Priority` attribute to `Medium`, the default priority when not set.
+         * - `'high'` will set the `Priority` attribute to `High`.
+         *
+         * More information about the different priority levels can be found in
+         * [the specification](https://tools.ietf.org/html/draft-west-cookie-priority-00#section-4.1).
+         *
+         * **Note** This is an attribute that has not yet been fully standardized, and may change in the future.
+         * This also means many clients may ignore this attribute until they understand it.
+         */
+        priority?: "low" | "medium" | "heigh" | undefined;
```
Comment (codershiba) [inline types/express-session/index.d.ts:249]:
```suggestion
        priority?: "low" | "medium" | "high" | undefined;
```

Comment (peterblazejewicz) [reply types/express-session/index.d.ts]: thanks!

**Review by ghost — APPROVED** [review]

**Comment by SalmaGhoneim** [comment]
@horiuchi Can we please get this PR in soon? :)

**Review by ganesh-globant — APPROVED** [review]

**Review by jakebailey — APPROVED** [review]

**Comment by peterblazejewicz** [comment]
Ready to merge

**Pull request merged**